### PR TITLE
Removed hardcoded strings in DeleteHelper

### LINKED ISSUE #3050: Remove hard coded strngs in delete helper
**Summary:** 
Delete reasons are hardcoded, they should go into strings.xml file.

### PR DESCRIPTION
**Pull request by PavelAplevich**
Fixes #3050
Welcome from Belarus!) I'm beginner in programming for Android. And this is my first open-source project.
I hope I can be helpful in this project.

Fixes hardcoded strings in delete/DeleteHelper.java

A small change in the class for the possibility of localization of strings. Adding these strings with localization in Russian.

Tested on Xiaomi Redmi4x and Nexus One(emulator) with API level 25 and 22.
Снимок экрана от 2019-07-01 17-55-28

photo_2019-07-01_17-58-57

### CODE REVIEW

File: app/src/main/res/values-ru/strings.xml [hunk 1]
```diff
@@ -547,4 +547,12 @@
   <string name="dialog_box_text_nomination">Почему %1$s должно быть удалено?</string>
   <string name="review_is_uploaded_by">%1$s загружено с помощью: %2$s</string>
   <string name="default_description_language">Язык описаний по умолчанию</string>
+  <string name="delete_reason_spam_selfie">Селфи</string>
```
Comment (neslihanturan) [inline app/src/main/res/values-ru/strings.xml]: Removing hard coded strings and inserting them into strings.xml is perfectly fine. But we don't edit language specific strings.xml files manually. We just edit the default one ( app/src/main/res/values/strings.xml) as you also did. For outher languages we have translator updates. Please revert changes in this file.

Comment (PavelAplevich) [reply app/src/main/res/values-ru/strings.xml]: Done

**Comment by neslihanturan** [comment]
Thanks @PavelAplevich and congrats for your first contribution! :tada:

**Comment by PavelAplevich** [comment]
@neslihanturan 
I will be glad to help you) Thank you

**Pull request merged**